prime-router/src/test/kotlin/ReceiverTests.kt

```diff
@@ -273,4 +274,11 @@ internal class ReceiverTests {
         val receiver = Receiver("elr", "IGNORE", Topic.COVID_19, CustomerStatus.INACTIVE, translatorConfig)
         assertThat(receiver.transportType.type).isEqualTo("NULL")
     }
+
+    @Test
+    fun `test MOJURO receiver timezone`() {
```

Comment:
Minor: test name typo.

Also, why is it testing `isNotEqualTo` if it's creating a time in MHT and comparing it to what appears to be the same datetime?

Reply:
Thank you very much, Andrew.